pom.xml

```diff
@@ -46,14 +46,60 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-  
+
+    <swagger-version>2.0.1</swagger-version>
+    <jackson-version>2.9.1</jackson-version>
+    <jetty-version>9.4.9.v20180320</jetty-version>
```

Comment:
property naming of versions is inconsisted, stick to using dashes (-) or dots (.) - but do not mix

Reply:
accepted